lib/mappers/__tests__/mapper.sketch.test.js

```diff
@@ -1,7 +1,12 @@
-const FIXTURES = require('./__fixtures__/mapper');
+const FIXTURES = require('./__fixtures__/mapper.sketch');
 
-const mapper = require('..');
-const formatColor = require('../color');
+const getMapper = require('..');
+const formatColor = require('../sketch/color');
+
+let mappers;
+beforeAll(() => {
+   mappers = getMapper("sketch");
```

Comment:
Same remark here, why is this not just a globally scoped const and is it being reassigned?